src/Contracts/Providers/Composer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * This file is part of Zero Framework.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace NunoMaduro\ZeroFramework\Contracts\Providers;
+
+/**
+ * The is the Zero Framework composer contract.
+ *
+ * @author Nuno Maduro <enunomaduro@gmail.com>
+ */
+interface Composer
+{
+    /**
+     * Pulls the provided package.
+     *
+     * @param  string $package
+     *
+     * @return $this
+     */
+    public function require(string $package): ComposerContract
```

Comment:
Bad return type?

Reply:
@GrahamCampbell Thanks for the return! Actually the previous PR was only a POC for @introwit.

Now the PR contain the real implementation. Please fell free to perform the review! 😊

You are more than welcome!